File: frontend/src/types/dentalService.ts
export interface DentalService {
  id: string;
  name: string;
  description: string | null;
  defaultPrice: number;
  durationMinutes: number;
  isActive: boolean;
  createdAtUtc: string;
  updatedAtUtc: string | null;
}

export interface CreateDentalServiceRequest {
  name: string;
  description?: string;
  defaultPrice: number;
  durationMinutes: number;
}

export interface UpdateDentalServiceRequest extends CreateDentalServiceRequest {}

File: backend/Entities/DentalService.cs
namespace ClinicFlow.Api.Entities;

public class DentalService
{
    public Guid Id { get; set; }
    public required string Name { get; set; }
    public string? Description { get; set; }
    public decimal DefaultPrice { get; set; }
    public int DurationMinutes { get; set; }
    public bool IsActive { get; set; } = true;
    public DateTime CreatedAtUtc { get; set; }
    public DateTime? UpdatedAtUtc { get; set; }
}
